apps/skus/src/components/SkuInfo.tsx

```diff
@@ -56,6 +56,13 @@ export const SkuInfo: FC<Props> = ({ sku = makeSku() }) => {
           </Text>
         </ListDetailsItem>
       ) : null}
+      {sku.hs_tariff_number != null ? (
+        <ListDetailsItem label='HS Code' gutter='none'>
```

Comment:
Maybe we have to reuse the same label as in the old admin, what do you think?

Reply:
We discussed together with Matteo and Simone that the HS Code label is fine here